packages/client/test/sync/lightsync.spec.ts

```diff
@@ -71,11 +72,16 @@ tape('[LightSynchronizer]', async (t) => {
       chain,
     })
     sync.best = td.func<typeof sync['best']>()
+    sync.latest = td.func<typeof sync['latest']>()
     td.when(sync.best()).thenReturn({ les: { status: { headNum: new BN(2) } } } as any)
+    td.when(sync.latest(td.matchers.anything())).thenResolve({ number: new BN(2) })
```

Comment:
Some annotation since I am also reading up stuff: [td.matchers.anything()](http://testdouble.github.io/testdouble.js/5-stubbing-results.html#tdmatchersanything)